RTN-004.tex

```diff
@@ -53,7 +52,312 @@
+Participants from the science community, "DP0 delegates", will be data rights holders\footnote{Data rights holders are US or Chilean scientists, or individuals associated with an International Contribution (in-kind) proposal in progress, as defined in \citeds{RDO-013}.} who represent a diversity of backgrounds.
+The term "delegates" reflects the fact that these individuals will have the important role of representing the science community as learners and as contributors to the overall success of DP0.
+The nature of these two goal-driven components of DP0 participation will vary according to individual delegates' interests and experience.
+
+\subsection{Two Stages: DP0.1 and DP0.2}\label{ssec:intro_stages}
+
+The simulated data set for DP0 was generated by the Dark Energy Science Collaboration (DESC) and is being used under an agreement between DESC and Rubin.
+This data set is the second in a series of data challenges, and is thus named DC2.
+More information about DC2 can be found in the DESC DC2 paper \citep{2020arXiv201005926L} and on the DESC website\footnote{\url{https://lsstdesc.org/DC2-production/Documents/}}.
+
+The DP0.1 data set will contain the DC2 images and catalogs processed by DESC.
+The DP0.1 data products will be available in the RSP in their original format (which is similar to planned LSST data products).
+The DP0.1 data set will begin to be available in the RSP starting June 30 2021.
+This will at first include a subset of the data products (processed visit images and visit-level catalogs without full-scale validation and quality analysis) and subsequently include science-ready data products.
+Therefore, in Section \ref{ssec:sel_time} the first stage of DP0 delegates are scheduled to have their accounts activated by June 30 2021.
+
+The DP0.2 data set will contain the DC2 images and catalogs reprocessed by the Rubin science pipelines.
+The DP0.2 data products will be available in the RSP in a format that is consistent with the planned LSST data products.
+The DP0.2 data set will begin to be available in the RSP starting September 30 2021.
+As with DP0.1, this will at first include a subset of the data products and subsequently include science-ready data products, which are projected to become available in the RSP starting March 31 2022.
+Therefore, in Section \ref{ssec:sel_time} the second stage of DP0 delegates are scheduled to have their accounts activated by March 31 2022, when \emph{all} DP0-related data products will be available.
+
+Further details about the DP0.1 and DP0.2 data sets will be presented elsewhere\footnote{Any references in other documents to "early access" data sets for DP0 refer to images and visit-level catalogs without full-scale validation and quality analysis. This is an internal term to guide Rubin Observatory's DP0-related development, and does not relate to "early access" for community participants.}.
```

Comment:
Early access has been changed in the milestones to be QA access, and this has been propagated to rtn-001. Maybe we can drop all mention now of "early access"

Reply:
Wonderful. Footnote deleted.